src/api/Shrooms.Premium.Tests/DomainService/Validators/LotteryValidatorTests.cs

```diff
@@ -0,0 +1,393 @@
+﻿using NSubstitute;
+using NUnit.Framework;
+using Shrooms.Contracts.DAL;
+using Shrooms.Tests.Extensions;
+using Shrooms.Contracts.Infrastructure;
+using Shrooms.DataLayer.EntityModels.Models;
+using System.Data.Entity;
+using Shrooms.Premium.Domain.DomainServiceValidators.Lotteries;
+using System;
+using System.Collections.Generic;
+using Shrooms.Contracts.DataTransferObjects;
+using Shrooms.Premium.Domain.DomainExceptions.Lotteries;
+using Shrooms.Premium.DataTransferObjects.Models.Lotteries;
+using Shrooms.DataLayer.EntityModels.Models.Lottery;
+using Shrooms.Contracts.Enums;
+
+namespace Shrooms.Premium.Tests.DomainService.Validators
+{
+    [TestFixture]
+    public class LotteryValidatorTests
+    {
+        private ISystemClock _systemClock;
+        private DbSet<ApplicationUser> _usersDbSet;
+
+        private LotteryValidator _sut;
```

Comment:
the name `_sut` sounds strange

Reply:
Means "System under test". I noticed some of the tests had that name, and it made more sense to me to call it that since it makes it easier to distinguish from other fields in the test class. Should I call it **_systemUnderTest** or just **_lotteryValidator** like I used to do?

Reply:
I think we can leave as is